app/_locales/hi/messages.json

```diff
@@ -2133,7 +2136,7 @@
     "message": "पर्याप्त गैस नहीं"
   },
   "notifications": {
-    "message": "सूचनाएं"
+    "message": "नोटीफिकेशन"
```

Comment:
erm... so idk Hindi. 🤷🏼‍♀️ I take it Crowdin found a more proper translation. The original translation was the output translating "Notifications" in Google Translate. Not sure why else this would change.

![hi-notifications](https://user-images.githubusercontent.com/20778143/174403167-d56c1212-970d-4253-bfe3-7af634d560df.gif)

Reply:
@digiwand  The new translation is just the English word `Notification` written in Hindi. The old translation seems to be the correct Hindi word; not sure why this has to be updated.

Reply:
@NiranjanaBinoy Thanks for the info here! 

@adonesky1 How should we proceed with this?

Reply:
Hmmm I'm a bit confused... What's wrong with the new translation? It's just less correct? @NiranjanaBinoy

Reply:
The word we had previously is the Hindi word for `Notification`, but the new translation is the word `Notification` written in Hindi.

Reply:
Interesting! Clearly I don't understand the nuances of Hindi, but I will ask our translations vendor about this!

Reply:
I've messaged them to take a look! Thanks for raising this @NiranjanaBinoy!

Reply:
I've just reverted this on the crowdin platform so it should be correct again once we get another push from crowdin!